devtools/aesh/src/main/java/io/quarkus/cli/commands/BuildToolConverter.java

```diff
@@ -0,0 +1,16 @@
+package io.quarkus.cli.commands;
+
+import org.aesh.command.converter.Converter;
+import org.aesh.command.converter.ConverterInvocation;
+
+import io.quarkus.devtools.project.BuildTool;
+
+public class BuildToolConverter implements Converter<BuildTool, ConverterInvocation> {
+    @Override
+    public BuildTool convert(ConverterInvocation invocation) {
+        if (invocation.getInput() != null && invocation.getInput().length() > 0)
+            return BuildTool.findTool(invocation.getInput());
+        else
+            return null;
```

Comment:
Maybe we should throw an exception here (instead of returning null)?

Reply:
Yes, probably wise. I didn't care as much as the null reference is handled by Æsh, but yeah. I'll change it.